### ISSUE TITLE
Automatically release translations in CI

### ISSUE THREAD

**Issue by tomasMizera** [MerginMaps/mobile#1884]
When transifex bot creates PR with new translation, let's add CI workflow that is automatically going to release the new translation and push it to the same branch.

In that case we will not need to release translations manually and users won't see failing job in readme.

**Comment by PeterPetrik** [comment]
Yep, this would be nice, so after merge of TX pull request the binaries are updated.
